posix-configs/eagle/200qx/px4-flight.config

```diff
@@ -0,0 +1,80 @@
+uorb start
+param set CAL_GYRO0_ID 2293760
+param set CAL_ACC0_ID 1310720
+param set CAL_ACC1_ID 1376256
+param set CAL_MAG0_ID 196608
+commander start
+param set RC_RECEIVER_TYPE 1
+rc_receiver start -D /dev/tty-1
+attitude_estimator_q start
+position_estimator_inav start
+mc_pos_control start
+mc_att_control start
+sleep 1
+param set SYS_AUTOSTART 4001
+param set SYS_AUTOCONFIG 1
+param set MAV_TYPE 2​
+param set RC_MAP_THROTTLE 1
+param set RC_MAP_ROLL 2
+param set RC_MAP_PITCH 3
+param set RC_MAP_YAW 4
+param set RC_MAP_MODE_SW 5
+param set RC_MAP_POSCTL_SW 6
+param set RC1_MAX 1900
+param set RC1_MIN 1099
+param set RC1_TRIM 1099
+param set RC1_REV 1
+param set RC2_MAX 1900
+param set RC2_MIN 1099
+param set RC2_TRIM 1500
+param set RC2_REV -1
+param set RC3_MAX 1900
+param set RC3_MIN 1099
+param set RC3_TRIM 1500
+param set RC3_REV 1
+param set RC4_MAX 1900
+param set RC4_MIN 1099
+param set RC4_TRIM 1500
+param set RC4_REV -1
+param set RC5_MAX 1900
+param set RC5_MIN 1099
+param set RC5_TRIM 1500
+param set RC5_REV 1
+param set RC6_MAX 1900
+param set RC6_MIN 1099
+param set RC6_TRIM 1099
+param set RC6_REV 1
```

Comment:
Can we get rid of the RC calibration parameters because we need to calibrate RC through QGC anyway? And the params are stored, right?